src/lib-components/BlockForm.vue

```diff
@@ -394,12 +646,80 @@ export default {
             }
         }
     }
+    .form-errors {
+        background-color: var(--color-status-error-01);
+        box-sizing: border-box;
+        padding: 20px;
+        border-radius: var(--rounded-slightly-all);
+    }
+}
+
+.submitButton {
+    box-sizing: border-box;
+    position: relative;
+    @include button;
+    min-height: 48px;
+    padding: 4px 40px;
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+    gap: 4px;
+    border: 1.5px solid var(--color-primary-blue-02);
+    transition-property: all;
+    @include animate-normal;
+    overflow: hidden;
+    z-index: 0;
+
+    background-color: var(--color-primary-blue-03);
+    --button-background-slide: var(--color-white);
+    border-color: var(--color-primary-blue-03);
+    color: var(--color-white);
 
-    button {
-        border: none;
+    .label {
+        white-space: nowrap;
     }
-    .form-errors {
-        background-color: cyan;
+
+    &::before {
+        content: "";
+        width: 100%;
+        height: 100%;
+        background-color: var(--button-background-slide);
+        position: absolute;
+        top: 0;
+        left: -100%;
+        transition-property: all;
+        @include animate-normal;
+        z-index: -10;
+    }
+
+    // Hover states
+    @media #{$has-hover} {
+        &:hover,
+        &:focus,
+        &:focus-visible {
+            cursor: pointer;
+            border-color: var(--color-primary-blue-02);
+            color: var(--color-black);
+
+            &::before {
+                left: 0;
+            }
+        }
+
+        &:focus,
+        &:focus-visible {
+            outline: none;
+            border-radius: 0;
+        }
+    }
+    // Breakpoints
+    @media #{$medium} {
+        padding: 4px 16px;
+        display: inline-flex;
+    }
+
+    @media #{$small} {
+        width: 100%;
```

Comment:
```
.registrationInfo {
flex-direction: column;
gap: 8px;
}
```

Reply:
Like this? 
![image](https://user-images.githubusercontent.com/47605309/195166439-4e9213f9-beb1-421c-a463-4286232b41bc.png)

Reply:
yes 💯